pages/assessment.tsx

```diff
@@ -50,10 +50,14 @@ const Assessment: NextPage = () => {
     'let - block scope. Dersom variabelen blir deklarert med let i en funksjon, er den bare tilgjengelig i funksjonen. var - global scope Dersom variabelen blir deklarert med var, blir den tilgjengelig i all kode. Kan by på problemer når vi gir variabler samme navn.';
```

Comment:
Did not do anything in this file, so this is all prettier....

Reply:
Super nice of you to point that out!